bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html

```diff
@@ -0,0 +1,240 @@
+            <h3>{{ _('Next-Gen CSS Engine') }}</h3>
+            <p class="cta"><a href="#nextgen" class="button button-hollow">{{ _('Learn more') }}</a></p>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="section section-newsletter">
+    <div class="content">
+    {% if LANG.startswith('en-') %}
+      {# L10n: line break for visual formatting #}
+      {{ email_newsletter_form('app-dev', 'Mozilla Developer Newsletter', subtitle=_('Get developer news, tricks and resources <br /> sent straight to your inbox.'), button_class='button-hollow button-light', include_language=False) }}
+    {% else %}
+      {{ email_newsletter_form(spinner_color='#fff', button_class='button-hollow button-light') }}
+    {% endif %}
+    </div>
+  </section>
+
+  <section class="section section-highlights">
+    <div class="content">
+
+      <div class="highlight-tools">
+        <h2 class="section-title"><span>{{ _('New Tools') }}</span></h2>
+
+        <div class="section-body">
+          <h3 id="devtools" class="body-title">{{ _('Firefox DevTools') }}</h3>
+
+          <img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-debugger-ani.gif') }}" alt="">
+          <p>
+          {% trans %}
+            The new Firefox DevTools are powerful, flexible, and best of all,
+            hackable. This includes a best-in-class JavaScript debugger, which
+            can target multiple browsers and is built in React and Ember.
+          {% endtrans %}
+        </div>
+      </div>
+
+      <div class="highlight-features">
+        <h2 class="section-title"><span>{{ _('Innovative Features') }}</span></h2>
+
+        <div class="section-body">
+          <h3 id="cssgrid" class="body-title">{{ _('Master CSS Grid') }}</h3>
+
+          <img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-cssgrid-ani.gif') }}" alt="">
+          <p>
+          {% trans %}
+            Firefox is the only browser with tools built specifically for
+            building and designing with CSS Grid. These tools allow you to
+            visualize the grid, display associated area names, preview
+            transformations on the grid and much more.
+          {% endtrans %}
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="section section-performance">
+    <div class="content">
+      <h2 id="nextgen" class="section-title"><span>{{ _('Faster Performance') }}</span></h2>
+
+      <div class="section-body">
+        <h3 class="body-title">{{ _('A Next-Generation CSS Engine (hint: it’s fast)') }}</h3>
+        <img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/stylo-engine.svg') }}" alt="">
+        <p>
+        {% trans %}
+          Firefox Quantum includes a new CSS engine, written in Rust, that has state-of-the-art innovations and is blazingly fast.
```

Comment:
Maybe we should put a max-width on this blurb. It's such a short sentence, I get "fast." on its own line in a wide window. It'll be different for each locale but I think something like `max-width: $width-phone-wide; margin: 0 auto 1.25em;` might not hurt.